internal/operators/ocs/ocs_operator.go

```diff
@@ -74,8 +74,35 @@ func (o *operator) ValidateCluster(_ context.Context, cluster *common.Cluster) (
 }
 
 // ValidateHost verifies whether this operator is valid for given host
-func (o *operator) ValidateHost(_ context.Context, _ *common.Cluster, _ *models.Host) (api.ValidationResult, error) {
-	return api.ValidationResult{Status: api.Success, ValidationId: o.GetHostValidationID(), Reasons: []string{}}, nil
+func (o *operator) ValidateHost(ctx context.Context, cluster *common.Cluster, host *models.Host) (api.ValidationResult, error) {
+	numOfHosts := len(cluster.Hosts)
+	if host.Inventory == "" {
+		o.log.Info("Empty Inventory of host with hostID ", host.ID)
+		return api.ValidationResult{Status: api.Pending, ValidationId: o.GetClusterValidationID(), Reasons: []string{"Missing Inventory in some of the hosts"}}, nil
```

Comment:
Please fix `GetClusterValidationID()` to `GetHostValidationID()`

Reply:
Done